pocs/scheduler/scheduler.py

```diff
@@ -175,8 +175,6 @@ def clear_available_observations(self):
         """Reset the list of available observations"""
         # Clear out existing list and observations
         self.current_observation = None
-        self._fields_file = None
```

Comment:
Oh, that looks like a problem. Why have you not seen the problem I reported? Are you restarting PAN001 most days?

Reply:
The `clear_available_observations` is triggered when no observations are found, which will happen if it is looking for a new target close to sunrise. PAN001 hasn't been operating most days since I started it because of bad weather (I've only had two nights in last week) and the other times it was in the middle of an observation all the way until Unsafe (because of daytime), so it never got to the NoObservations from scheduling. (This last part is vaguely related to the first checkbox in #361, where it is scheduling a sequence even though the tail end gets interrupted by either meridian or sunrise.)